ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/util/MigrationsDirectoryUtil.java

```diff
@@ -99,4 +94,34 @@ public static String computeHashForFile(final String filename) {
           "Could not compute hash for file '%s': %s", filename, e.getMessage()));
     }
   }
+
+  public static String getNameFromMigrationFilePath(final String filename) {
+    return filename
+        .substring(filename.indexOf("__") + 2, filename.indexOf(".sql"))
+        .replace('_', ' ');
+  }
+
+  public static int getVersionFromMigrationFilePath(final String filename) {
+    return Integer.parseInt(filename.substring(1, 7));
```

Comment:
We should validate the filename format, rather than letting this throw if this particular substring is not an integer. If we detect files that don't fit the expected format (`V<six digit number>__<name>.sql`), we should log a warning saying that those files are being ignored.